common/JackGraphManager.cpp

```diff
@@ -24,7 +24,11 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
 #include <assert.h>
 #include <stdlib.h>
 #include <algorithm>
+#ifdef _WIN32
```

Comment:
Maybe change all of these to _MSC_VER.

Reply:
most of these were `ifdef WIN32` before I changed them too `ifdef _WIN32`, how does mingw tread the ifdef WIN32 that are currently on develop branch? Will this not break mingw?